### ISSUE TITLE
Update Faraday in Gemspec

### ISSUE THREAD

**Issue by oyeanuj** [metainspector/metainspector#256]
Hi @jaimeiniesta, it seems like while trying to upgrade to Ruby 2.7, Faraday needs to be [updated to 1.0](https://github.com/lostisland/faraday/blob/master/CHANGELOG.md) but MetaInspector specifies ~0.17 for Faraday.

Any plans on making the gemspec requirement accommodate Faraday 1.0?

**Comment by jschwindt** [comment]
We need to wait until `faraday_middleware v1.0.0` is also shipped, see: https://github.com/lostisland/faraday_middleware/issues/200

**Comment by mindtonic** [comment]
@oyeanuj Thanks for all you guy do, we really appreciate it! 

It looks like `faraday_middleware v1.0.0` shipped 11 days ago. I know the world is crazy now but I was curious if you had any ideas on an update timeline to accommodate Faraday 1.0?

**Comment by jschwindt** [comment]
I'll take care of it and let you know, thanks for the heads up!

**Comment by jschwindt** [comment]
Still having dependency problems, this time with `faraday-http-cache`:

```
  > bundle
  Fetching gem metadata from http://rubygems.org/.........
  Resolving dependencies...
  Bundler could not find compatible versions for gem "faraday":
    In Gemfile:
      metainspector was resolved to 5.9.0, which depends on
        faraday-http-cache (~> 2.1.0) was resolved to 2.1.0, which depends on
          faraday (~> 0.8)

      metainspector was resolved to 5.9.0, which depends on
        faraday (~> 1.0.0)
```
Not sure why is it happening because `faraday-http-cache` correctly depends of `faraday >= 0.8` but bundler whats ~> 0.8

**Comment by bricemaurin** [comment]
hi @jschwindt did you succeed updating this ? thanks 🙏 in advance !

**Comment by jschwindt** [comment]
faraday-http-cache was updated a few dates ago, I'll give it a try now! Thanks!

**Comment by jaimeiniesta** [comment]
MetaInspector 5.10.0 has been released, with support for Faraday 1.0
